python/ray/tune/logger.py

```diff
@@ -16,6 +16,11 @@
     tf = None
     print("Couldn't import TensorFlow - this disables TensorBoard logging.")
 
+try:
```

Comment:
this could go in a different PR

Reply:
Don't we have yaml as a Ray dep now, so it will be installed by default?